config/core/test-configs.yaml

```diff
@@ -2384,6 +2390,7 @@ test_configs:
       - baseline
       - baseline-nfs
       - cros-ec
+      - iec-security
```

Comment:
Maybe run it on `mt8173-elm-hana` as it has more online devices in Collabora's lab (10 hana, whereas 5 kevin).  They are both arm64 Chromebooks.

Reply:
added

Reply:
I meant, `mt8173-elm-hana` _instead of_ `rk3399-gru-kevin`.

Reply:
fixed